app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -76,6 +79,8 @@ private void initList() {
 
         if (!isInEditMode()) {
             SearchTimeline timeline = new SearchTimeline.Builder()
+                    .resultType(SearchTimeline.ResultType.RECENT)
+                    .maxItemsPerRequest(10)
```

Comment:
We went for 10 because our viewholder is quite large and there's already some scrolling to do before reaching the end. I'd say we can go for 20 or 30 (which is the default value)